src/openforms/appointments/base.py

```diff
@@ -4,14 +4,14 @@
 from typing import Dict, List, Optional
 
 from django.urls import reverse
-from django.utils.translation import gettext_lazy as _
 
 from rest_framework import serializers
 
 from openforms.submissions.models import Submission
 from openforms.utils.mixins import JsonSchemaSerializerMixin
 from openforms.utils.urls import build_absolute_uri
 
+from ..plugins.plugin import AbstractPlugin
```

Comment:
please use absolute imports when importing outside of the django app/openforms module